src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -514,12 +515,29 @@ protected function initializeContainer()
         $cache = new ConfigCache($this->getCacheDir().'/'.$class.'.php', $this->debug);
         $fresh = true;
         if (!$cache->isFresh()) {
-            $container = $this->buildContainer();
+            if ($this->debug) {
+                $collectedLogs = array();
+                $previousHandler = set_error_handler(function () use (&$collectedLogs, &$previousHandler) {
+                    $collectedLogs[] = $args = func_get_args();
+
+                    return $previousHandler;
```

Comment:
Instead of display the sum of those logs, we can display the regular logs handled, and if there isn't, display the "boot logs" counter ?